components/navbar/ContributeButton.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+import Link from "next/link";
+import { AiFillGithub } from "react-icons/ai";
+
+const ContributeButton = () => {
+  return (
+    <Link
+      href="https://github.com/coronasafe/leaderboard/issues"
```

Comment:
Leaderboard would be deployed by different organizations, it'd be great to open the organization as the viewer might be looking to contribute to the organization.

Organization-specific data is not kept in this repo.
Use environment variables.

Reply:
ok, I will update it with .env varible